src/cpu/cpu_convolution_list.cpp

```diff
@@ -225,6 +226,7 @@ const std::map<pk_dt_impl_key_t, std::vector<impl_list_item_t>> &impl_list_map()
             CPU_INSTANCE_AARCH64(jit_uni_dw_convolution_fwd_t<sve_256, bf16, bf16>)
             CPU_INSTANCE_AARCH64(brdgmm_dw_convolution_fwd_t<sve_256>)
             CPU_INSTANCE_AARCH64(brdgmm_dw_convolution_fwd_t<sve_128>)
+            CPU_INSTANCE_AARCH64(jit_uni_dw_convolution_fwd_t<sve_128, bf16, bf16>)
```

Comment:
I'm not sure I understand where the third parameter comes from given that instantiations are only done for two parameters... Any insights on that?

Reply:
Default it is, though it better to disappear... Thanks for clarifying.